site/_docs/history.md

```diff
@@ -29,25 +29,114 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / XXXX-XX-XX
-{: #v1-37-0}
 
-This release comes x months after [1.36.0](#v1-36-0),
-contains contributions from x contributors, and resolves x issues.
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.38.0">1.38.0</a> / XXXX-XX-XX
+{: #v1-38-0}
 
+This release comes x months after [1.37.0](#v1-37-0),
+contains contributions from x contributors, and resolves x issues.
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-38-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 21.0 to 32.1.3-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-38-0}
+
+#### Dependency version upgrade
+{: #dependency-1-38-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-38-0}
+
+#### Build and test suite
+{: #build-1-38-0}
+
+#### Web site and documentation
+{: #site-1-38-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / 2024-23-04
```

Comment:
I think the date is in the wrong format (it should be YYYY-MM-DD)

Reply:
yep, compared with previous records and you are right
I also replaced `XXXX-XX-XX` with `YYYY-MM-DD` to avoid this issue in future